.github/scripts/crib/pr-comment-crib-env.js

```diff
@@ -19,9 +34,19 @@ async function commentExists(octokit, owner, repo, prNumber, uniqueIdentifier) {
 async function run() {
   try {
     const token = process.env.GITHUB_TOKEN;
-    const octokit = new Octokit({ auth: token });
+    const route53ZoneId = process.env.ROUTE53_ZONE_ID;
+
+    // Check for the existence of GITHUB_TOKEN and ROUTE53_ZONE_ID
+    if (!token || !route53ZoneId) {
+      core.setFailed("Error: Missing required environment variables.");
+      if (!token) core.setFailed("GITHUB_TOKEN is required.");
+      if (!route53ZoneId) core.setFailed("ROUTE53_ZONE_ID is required.");
+      return;
```

Comment:
Might be better to throw an error instead

Reply:
Revamped in dffc5914d477a8afc2f5c8d060df203e83eef8e9 -- didn't throw because this is in the main `run()` function.